types/types.go

```diff
@@ -198,6 +198,7 @@ type Info struct {
 	Images             int
 	Driver             string
 	DriverStatus       [][2]string
+	CustomStatus       [][2]string
```

Comment:
I'll update the PR to `SystemStatus`, but it would nice to have some comments that specify what it does. Is there a recommended location for that, because none of the other fields have attached comments.